agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -276,6 +276,16 @@ bool backhaul_manager::socket_disconnected(Socket *sd)
                 finalize_slaves_connect_state(true, slaves_sockets.front());
             }
             LOG(INFO) << "disconnected slave sockets has been deleted";
+
+            if (m_eFSMState >= EState::CONNECT_TO_MASTER) {
```

Comment:
Since it's not to controller (see below) it should be any time that the slaves are connected.

Reply:
Yes, the 1905 standard does not say it is for the controller but the reason for that is that IEEE 1905.1 does not define entities such as the "Controller" and "Agent". But the whole point of the message is that the controller will receive it.
So there is no point to send it if the controller will not receive it.

Reply:
Not necessarily. An agent can also do topology queries. That can be used for keeping track of the backhaul link (#951) or for agent-initiated steering.